app/models/miq_request.rb

```diff
@@ -237,8 +237,7 @@ def v_approved_by
   end
 
   def v_approved_by_email
-    emails = miq_approvals.inject([]) { |arr, a| arr << a.stamper.email unless a.stamper.nil? || a.stamper.email.nil?; arr }
-    emails.join(", ")
+    miq_approvals.collect { |i| i.stamper.try(:email) }.compact.join(", ")
```

Comment:
I thought about adding tests for it, but it is actually tested already

https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_request_spec.rb#L201
https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_request_spec.rb#L237
https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_request_spec.rb#L245